# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../cdrtools/linux26-types.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./libscg/scsi-linux-sg.c.orig	2002-10-21 19:20:35.000000000 +0300
+++ ./libscg/scsi-linux-sg.c	2004-04-12 12:26:37.000000000 +0300
@@ -66,6 +66,11 @@
 #if LINUX_VERSION_CODE >= 0x01031a /* <linux/scsi.h> introduced in 1.3.26 */
 #if LINUX_VERSION_CODE >= 0x020000 /* <scsi/scsi.h> introduced somewhere. */
 /* Need to fine tune the ifdef so we get the transition point right. */
+#if LINUX_VERSION_CODE >= 0x020600
+#define __KERNEL__
+#include <asm/types.h>
+#undef __KERNEL__
+#endif
 #include <scsi/scsi.h>
 #else
 #include <linux/scsi.h>
